crates/swc_ecma_minifier/tests/terser/compress/blocks/issue_1672_if_strict/output.js

```diff
@@ -1,12 +1,10 @@
 "use strict";
-switch (
-    function () {
-        return xxx;
-    }
-) {
+switch(function() {
+    return xxx;
+}){
     case xxx:
         if (console.log("FAIL")) {
-            function xxx() {}
+            function xxx1() {}
         }
```

Comment:
This is because swc minifier test defaults to `Module`, but does this really matter?